File: Assignment-33/Program3.java
import java.util.*;

class Program3
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Number:");
            int iNo = sobj.nextInt();

            Digit obj = new Digit();

            int iRet = obj.CountRange(iNo);
            System.out.println(iRet);
      }
}
class Digit
{
      public int CountRange(int iNo)
      {
            int iCnt = 0;
            int iDigit = 0;

            if(iNo < 0)
            {
                  iNo = -iNo;
            }
            while(iNo !=0)
            {
                  iDigit = iNo % 10;
                  
                  if((iDigit > 3)&&(iDigit < 7))
                  {
                        iCnt++;
                  }

                  iNo = iNo / 10;
            }
            return iCnt;

      }
}

File: Assignment-13/program3.c
#include<stdio.h>

int CountRange(int iNo)
{
      int iDigit = 0;
      int iCnt = 0;

      while(iNo!=0)
      {
            iDigit = iNo % 10;

            if((iDigit > 3) && (iDigit < 7))
            {
                  iCnt ++;
            }

            iNo = iNo / 10;
      }
      return iCnt++;
}
int main()
{
      int ivalue = 0;
      int iRet = 0;

      printf("Enter The Number :");
      scanf("%d",&ivalue);

      iRet = CountRange(ivalue);

      printf("%d",iRet);

      return 0;
}